testing/trino-product-tests/src/main/java/io/trino/tests/product/iceberg/TestIcebergRedirectionToHive.java

```diff
@@ -445,23 +469,15 @@ public void testSystemJdbcColumns()
         assertThat(onTrino().executeQuery(
                 format("SELECT table_cat, table_schem, table_name, column_name FROM system.jdbc.columns WHERE table_cat = 'iceberg' AND table_schem = '%s' AND table_name = '%s'", schemaName, tableName)))
                 .containsOnly(
-                        // TODO report table's columns via redirect to Hive
-//                        row("iceberg", schemaName, tableName, "nationkey"),
-//                        row("iceberg", schemaName, tableName, "name"),
-//                        row("iceberg", schemaName, tableName, "comment"),
-//                        row("iceberg", schemaName, tableName, "regionkey")
-                        /**/);
+                        row("iceberg", schemaName, tableName, "nationkey"),
+                        row("iceberg", schemaName, tableName, "name"),
+                        row("iceberg", schemaName, tableName, "comment"),
+                        row("iceberg", schemaName, tableName, "regionkey"));
 
-        // test via redirection with just schema filter
+        // test via redirection with just schema filter - consistent with the functionality of the command `SHOW TABLES` command on the Iceberg connector
```

Comment:
Note the change in functionality here for the Iceberg connector. Please provide feedback whether the provided functionality is incorrect from your perspective.